### ISSUE TITLE
Add license metadata for MP to OBO Foundry

### ISSUE THREAD

**Issue by cmungall** [obophenotype/mammalian-phenotype-ontology#2252]
The subject of license for MP came up on twitter: https://twitter.com/GreeneScientist/status/877887237878185985

In OLS it says "open source license pending". I think that means that the field is not filled in our metadata, rather than say waiting on approval by lawyers or something like that: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/mp.md (@SimonJupp, maybe this could be clearer in OLS?)

@sbello and @cindyJax - do you want to select a license? You can continue to leave this unspecified of course, but then people expect the most restrictive and will not use it.

Most people opt for CC-BY, but there is a movement to go with CC-0 (DO is doing this). You can see some discussion here:
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285

**Comment by sbello** [comment]
I've replaced the pending statement with the MGI copyright URL in the owl file.

**Comment by dhimmel** [comment]
For reference, here's the license text at http://www.informatics.jax.org/mgihome/other/copyright.shtml:

> THE JACKSON LABORATORY MAKES NO REPRESENTATION ABOUT THE SUITABILITY OR ACCURACY OF THIS SOFTWARE OR DATA FOR ANY PURPOSE, AND MAKES NO WARRANTIES, EITHER EXPRESS OR IMPLIED, INCLUDING MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE OR THAT THE USE OF THIS SOFTWARE OR DATA WILL NOT INFRINGE ANY THIRD PARTY PATENTS, COPYRIGHTS, TRADEMARKS, OR OTHER RIGHTS. THE SOFTWARE AND DATA ARE PROVIDED "AS IS".

> This software and data are provided to enhance knowledge and encourage progress in the scientific community and are to be used only for research and educational purposes. Any reproduction or use for commercial purpose is prohibited without the prior express written permission of the Jackson Laboratory.

> Copyright © 1996, 1999, 2002, 2005, 2008, 2011, 2012, 2015 by The Jackson Laboratory
> All Rights Reserved

The license doesn't grant users any rights. So unless one feels the database is not subject to copyright or that their use is "fair use", then any reuse is copyright infringement. By not having an open license, you're exposing your users to legal risks for their use of this publicly funded resource. It's not nice.

@sbello are you still considering switching to an [open license](http://opendefinition.org/licenses/)?

**Comment by cmungall** [comment]
I hope the issue of MP licensing can be separated from the other MGI products...

**Comment by cindyJax** [comment]
@dhimmel You are misinterpreting the current license statement.  Only commercial use of the data is restricted.  Non-profit research institutions are free to use MGI's data without restriction.
@cmungall We are reviewing the possiblility of more explicit and less restricitve licensing for multiple components of MGI.

**Comment by dhimmel** [comment]
> Non-profit research institutions are free to use MGI's data without restriction.

@cindyJax, where does the MGI statement explicitly grant users from non-profit research institutions any rights?

Since "all rights reserved" is the default copyright status, you need to explicitly permit usage. For example, the CC BY-NC license is one way to enable reuse with attribution as long as its non commercial. You can see from the [license text](https://creativecommons.org/licenses/by-nc/4.0/legalcode) that granting users reuse rights is actually a quite involved and legally complex process.

I'm not a lawyer, but I think a user would be foolish to interpret the current MGI statement as granting any rights. The statement does say

> This software and data are ... to be used only for research and educational purposes.

However, I don't read this as an affirmative grant of reuse rights. Certainly, it's sufficiently vague to provide little to no legal commitment on the part of MGI.

**Comment by cindyJax** [comment]
@ dhimmel As I said to @cmungall, our copyright statement and licensing are currently under review. It's a complex matter because MGI is a site that integrates data from dozens of sources in addition to our expert curation.  We cannot reissue data from other sources under a less restrictive license than the original source. 
I will post here when any changes are issued to our statements.

**Comment by dhimmel** [comment]
> I will post here when any changes are issued to our statements.

Great thanks!

> We cannot reissue data from other sources under a less restrictive license than the original source.

I completely understand! This is the crux of the problem. Without open licensing, integrating resources quickly becomes legally intractable. As publicly funded creators, it's our duty to try to prevent legal obstacles from compounding. I know that changes require lot's of review and should be made carefully.

So I guess the immediate question would be, is the Mammalian Phenotype Ontology entirely original work? If so, you could license MP more definitively, while the intricacies of other MGI resources are still being worked out. In the end, it may make the most sense to have separate licenses for separate MGI resources, especially if only some of those resources are encumbered with upstream licensing arrangements.
